go/ct/spc/specification.go

```diff
@@ -2102,19 +2110,29 @@ func callEffect(s *st.State, addrAccessCost vm.Gas, op OpCode) {
 		}
 	}
 
+	sender := s.CallContext.CallerAddress
```

Comment:
This changes the sender from the `AccountAddress` to the `CallerAddress` for regular calls and static calls. This is likely wrong and should be detected as a difference in the specification and the geth implementation.